README.md

```diff
@@ -551,7 +551,7 @@ ort:
 An AWS S3 Bucket can be used to store scan results. You must provide a previously created Bucket, an optional AWS region
 (if not present it will use [default](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/region-selection.html)) 
 and credentials (if not provided, system [default](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/credential-providers.html) 
-values will be used).
+values will be used). Optionally, you can set the data to be stored compressed.
```

Comment:
The commit message should probably also mention that compression is optional.

Reply:
Addressed by squashing.